config/main.py

```diff
@@ -1237,18 +1237,29 @@ def save(filename):
     if filename is not None:
         cfg_files = filename.split(',')
 
-        if len(cfg_files) != num_cfg_file:
+        # If only one filename is provided in multi-ASIC mode,
+        # save all ASIC configurations to that single file.
+        if len(cfg_files) == 1 and multi_asic.is_multi_asic():
```

Comment:
in this case for multi-asic, when single file name is provided to do `config save`, we are saving to 1 single file:
1. in this single file, do we have duplicate content from both host and asics?
2. what is the consequences of overwriting all config in one file? should we prevent this behavior

Reply:
1. We do have duplicate content. Just like we have duplicate content in current `config_db.json,config_db0.json,config_dbN.json`.
2. This is by design. NDM golden config for multi-asic will be single file also. Please check the HLD design doc.https://github.com/sonic-net/SONiC/pull/1684

Reply:
I see.
how do we interpret back the single file, into config dbs?

Reply:
I will create another PR to extend `config reload` CLI to interpret it.

Reply:
sounds good. please paste the other PR once ready. thanks!